src/shared/components/CommentSection.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from "react";
+import {
+  Box,
+  List,
+  Typography,
+} from "@mui/material";
+import useTheme from "features/theme/useTheme";
+import axios from "axios";
+import BaseComment from "features/comments/components/Comment";
+import { BaseCommentModel } from "features/comments/models/Comments.model";
+
+function CommentSection({ id }: any) {
```

Comment:
do we want id to be "any" type? 
or maybe
type CommentSectionProps = {
      id: number; 
}